.github/workflows/binaries-linux-arm32v7.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           docker cp $container_id:/root/hledger tmp
           docker rm -v $container_id
           cd tmp
-          tar cvf hledger-linux-arm32v7.tar hledger hledger-ui hledger-web
+          tar cvf hledger-linux-arm32v7.tar hledger hledger-ui hledger-web hledger/shell-completion/hledger-completion.bash
```

Comment:
This looks wrong - is `hledger` the binary file or a directory here ?

Reply:
Was meant to be the directory

Reply:
@hseg, all the other changes look right, but am I right in thinking this one is still wrong ? We rarely use this workflow, but as written it seems to treat `hledger` as both the binary and the directory at the same time.